MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -932,33 +880,37 @@ private int GetDefaultAudioBitrateIfUnknown(MediaStream audioStream)
             return 192000;
         }
 
-        private int GetAudioBitrate(string subProtocol, long maxTotalBitrate, string[] targetAudioCodecs, MediaStream audioStream, StreamInfo item)
+        private static int GetAudioBitrate(string subProtocol, long maxTotalBitrate, string[] targetAudioCodecs, MediaStream audioStream, StreamInfo item)
         {
-            var targetAudioCodec = targetAudioCodecs.Length == 0 ? null : targetAudioCodecs[0];
+            string targetAudioCodec = targetAudioCodecs.Length == 0 ? null : targetAudioCodecs[0];
 
-            var targetAudioChannels = item.GetTargetAudioChannels(targetAudioCodec);
+            int? targetAudioChannels = item.GetTargetAudioChannels(targetAudioCodec);
 
-            int defaultBitrate = audioStream == null ? 192000 : audioStream.BitRate ?? GetDefaultAudioBitrateIfUnknown(audioStream);
+            int defaultBitrate;
+            int encoderAudioBitrateLimit = int.MaxValue;
 
-            // Reduce the bitrate if we're downmixing
-            if (targetAudioChannels.HasValue && audioStream != null && audioStream.Channels.HasValue && targetAudioChannels.Value < audioStream.Channels.Value)
+            if (audioStream == null)
             {
-                defaultBitrate = targetAudioChannels.Value <= 2 ? 128000 : 192000;
+                defaultBitrate = 192000;
             }
-
-            int encoderAudioBitrateLimit = int.MaxValue;
-
-            if (audioStream != null)
+            else
             {
+                if (targetAudioChannels.HasValue && audioStream.Channels.HasValue && targetAudioChannels.Value < audioStream.Channels.Value)
+                {
+                    // Reduce the bitrate if we're downmixing
+                    defaultBitrate = targetAudioChannels.Value <= 2 ? 128000 : 192000;
```

Comment:
Let's not hurt stereo with 128kbps :)
```suggestion
                    defaultBitrate = targetAudioChannels.Value < 2 ? 128000 : 192000;
```